build/integration.sh

```diff
@@ -14,6 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+if [[ ! -z "${JENKINS_HOME-}" ]]
+then 
+    exec ./jenkins_e2e.sh
```

Comment:
Don't think so - exec replaces current shell process & doesn't return to caller iirc.

Not sure why you need to exec it though? Just call it normally? Or are you relying on the exit code of the exec script?